mixer/adapter/signalfx/cumulative.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2018 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package signalfx
+
+import (
+	"sync/atomic"
+
+	"github.com/signalfx/golib/datapoint"
+	"github.com/signalfx/golib/sfxclient"
+)
+
+// A CumulativeCollector tracks an ever-increasing cumulative counter
+type CumulativeCollector struct {
```

Comment:
Any specific reason why this is a public type? That is, why the uppercase C?

Reply:
I had originally added it in our golib where it was exported but moved it into here for now, so I'll change it to unexported.